### PR TITLE
Add support for multiples IaC sources

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | yes
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #50 
| ❓ Documentation  | yes

## Description

Support multiples `--from` flags as IaC source

### CODE REVIEW

File: pkg/remote/aws/test/route_table/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
```
Comment (moadibfr) [inline pkg/remote/aws/test/route_table/.terraform.lock.hcl:1]: is this file required ?

**Review by moadibfr — CHANGES_REQUESTED** [review]

File: pkg/remote/aws/test/route_table/.terraform.lock.hcl (hunk 1 above)
Comment (eliecharra) [reply pkg/remote/aws/test/route_table/.terraform.lock.hcl]: Of course not nice catch, I removed it

File: doc/cmd/scan/iac_source.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
 Currently, driftctl only supports reading IaC from a Terraform state.
 We are investigating to support the Terraform code as well, as a state does not represent an intention.
 
+ℹ️ Multiples states could be read by passing multiples flags `--from`
```
Comment (wbeuil) [inline doc/cmd/scan/iac_source.md:6]: `Multiple states can be read by passing --from flags`

**Review by wbeuil — COMMENTED** [review]
Wonderful work !

File: pkg/resource/chain_supplier.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+)
+
+type ChainSupplier struct {
+	suppliers []Supplier
+	runner    *parallel.ParallelRunner
+}
+
+func NewChainSupplier() *ChainSupplier {
+	return &ChainSupplier{
+		runner: parallel.NewParallelRunner(context.TODO(), 3),
```
[the remark this answers is not in the document]
Comment (eliecharra) [reply pkg/resource/chain_supplier.go:16]: Hmm good question, I put 3 in a complete arbitrary way. Maybe we could set `runtime.NumCPU()` ?

[the remark this answers is not in the document]
Comment (wbeuil) [reply pkg/resource/chain_supplier.go:16]: Maybe @moadibfr could help on this one

[the remark this answers is not in the document]
Comment (eliecharra) [reply pkg/resource/chain_supplier.go:16]: As we are only doing I/O read and processing, I think it make sense to use every CPU thread for state parsing.

[the remark this answers is not in the document]
Comment (wbeuil) [reply pkg/resource/chain_supplier.go:16]: Makes sense 👍🏻

**Review by wbeuil — DISMISSED** [review]

**Pull request merged**